### PR TITLE
Visual Novel

### PR DESCRIPTION
**Pull request by emily-j-liu**
Thanks for making a PR! Please fill in the below:

Name: Emily Liu

Do you run a club? Yes

What relevant experiences do you have to help you write this Jam? Experience with utilizing Renpy to create my own visual novels, an understanding of the features that it offers

Will you be available on Slack to provide support to Hack Clubbers who run into issues while going through your Jam? Yes

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+Name: Emily Liu
 
-Do you run a club? 
+Do you run a club? Yes
 
-What relevant experiences do you have to help you write this Jam?
+What relevant experiences do you have to help you write this Jam? Experience with utilizing Renpy to create my own visual novels, an understanding of the features that it offers
 
-Will you be available on Slack to provide support to Hack Clubbers who run into issues while going through your Jam?
+Will you be available on Slack to provide support to Hack Clubbers who run into issues while going through your Jam? Yes
```
Comment (zeyuyaoy) [inline pull_request_template.md:9]: You don't need to change this :)

File: jams/VN.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+### Details:
+_Outcome:_ Create an visual novel style game with interactive choices and different outcomes based on the choices made.
+
+_Requirements:_ Ren'Py 8.1.1
+
+_Language used:_ Renpy/Python
+
+_Customization opportunity:_ Jammers will be able to choose the genre, plot, and overall story flow of the visual novel, making it as complciated or as simple as they wish. 
+
+_Platform Limitations:_ Requires the installation of Renpy (Andriod/Linux/Windows/OSX/iOS)
```
Comment (zeyuyaoy) [inline jams/VN.md:12]: I don't think it'll qualify for the bounty, as it's only for PRs that can be done on the web (targeting Chromebooks).

File: jams/VN.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+Jammers have free reign to choose what kind of storyline they wish to create, and what genre the story falls under. They can create their own unique story with new characters, sprites, and varying levels of complexity.
+
+**Why should Hack Clubbers care about this project?**
+
+Renpy is a simple but powerful allowing Jammers to quickly create a simple visual novel, while also giving them the ability to add on more layers of complexity with additional Python knowledge.
+
+**What Club Members will walk away with (both in terms of knowledge and in terms of product)**
+
+An understanding of how Renpy works, an understanding of how to put together a basic visual novel using Renpy, and a simple visual novel that they created.
```
Comment (zeyuyaoy) [inline jams/VN.md:34]: Wild idea: You make this part of the workshop where you explain how Renpy works using Renpy :p

**Review by zeyuyaoy — CHANGES_REQUESTED** [review]
Left a few comments; otherwise, this PR looks super cool!

**Comment by sahitid** [comment]
sorry! i don't think we can approve this jam, due to the fact that it's not web based. have a great day!

**Pull request closed without merging**